src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -912,6 +912,44 @@ private void SpillEvalStack()
             }
         }
 
+        private void PushArray<T>(ImmutableArray<T> array, Func<T, IOperation> unwrapper = null) where T : IOperation
+        {
+            Debug.Assert(unwrapper != null || typeof(T) == typeof(IOperation));
+            foreach (var element in array)
+            {
+                _evalStack.Push(Visit(unwrapper == null ? element : unwrapper(element)));
+            }
+        }
+
+        private ImmutableArray<T> PopArray<T>(ImmutableArray<T> originalArray, Func<IOperation, int, ImmutableArray<T>, T> wrapper = null) where T : IOperation
```

Comment:
Are there any callers that pass `null` for `unwrapper` or `wrapper`? If not, consider removing default value and `null` check.

Reply:
Not yet, but there will be a few that will have `ImmutableArray<IOperation>` as their type.